### ISSUE TITLE
make local failed

### ISSUE THREAD

**Issue by yoichio** [whatwg/fetch#1198]
```
$ make local
bikeshed spec fetch.bs fetch.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
FATAL ERROR: Couldn't find 'RFC8941' in bibliography data. Did you mean:
  rfc2119
  secg1
  bocu1
  icc42
  rdfcal
 ✘  Did not generate, due to fatal errors
make: *** [Makefile:19: local] Error 2
```
It worked at 4202f5c
I guess https://github.com/whatwg/fetch/commit/873a42e746b5b33e5ac7a15364a1cdf72a05e200 is the culprit.

**Comment by annevk** [comment]
Does it still fail if you run `bikeshed update` first?

**Comment by yoichio** [comment]
Yes
```
$ bikeshed update
Updating via manifest...
Deleted 85 old data files.
Updating 1055 files...
Updated 358/1055...
Updated 999/1055...
Done!
$ make local
bikeshed spec fetch.bs fetch.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
FATAL ERROR: Couldn't find 'RFC8941' in bibliography data. Did you mean:
  rfc2119
  rec-xml
  pkcs1
  rdfcal
  csp1
 ✘  Did not generate, due to fatal errors
make: *** [Makefile:19: local] Error 2
```

**Comment by annevk** [comment]
That's weird, it's working fine here and it's working on the server as well as otherwise https://github.com/whatwg/fetch/commits wouldn't be green.

@tabatkins any idea what this could be?

**Comment by sideshowbarker** [comment]
Works fine for me too in my environment — both with pip3-installed Bikeshed, and with Bikeshed installed from the sources in the GH repo.

@yoichio Can you make try doing this:

    bikeshed update --biblio --skip-manifest

That’ll re-download the biblio data. And if that doesn’t fix it, maybe try:

    bikeshed update --skip-manifest

…which will re-download *all* the data files

**Comment by tabatkins** [comment]
No clue what might be happening here, `rfc8941` is definitely in the current set of data files.

**Comment by yoichio** [comment]
`bikeshed update --biblio --skip-manifest` fixed this! Thanks.
However it left some `LINK ERROR`s:
```
$ make local
bikeshed spec fetch.bs fetch.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
LINK ERROR: No 'dfn' refs found for 'download the hyperlink'.
<a data-lt="download the hyperlink" data-link-type="dfn">downloads</a>
LINK ERROR: No 'dfn' refs found for 'exclude fragment'.
<a data-link-for="URL serializer" data-link-type="dfn" data-lt="exclude fragment"><i>exclude fragment</i></a>
...
```
and full `bikeshed update --skip-manifest` fixed that all.

Should we update Makefile to check biblio data up-to-date?

**Comment by annevk** [comment]
If there's an easy cheap check we could add it to https://github.com/whatwg/spec-factory I suppose.

**Comment by tabatkins** [comment]
I'm confused that a --skip-manifest did something; it shouldn't have. It just does the full slow download of all the original sources, which is what [bikeshed-data](https://github.com/tabatkins/bikeshed-data) does to assemble the files that you download during a normal update.

I *suspect* that @yoichio might be on a slightly older Bikeshed version; I had a bug earlier this year where the manifest handling was incorrect and it could get stuck in an inconsistent state that made it *think* files were up-to-date when they weren't. That was fixed a while back, tho, in the last release I made to Pip.

Ignoring the possibility of a buggy old Bikeshed version, tho, we could just add a `bikeshed update` to the spec generation; it would add a network roundtrip to check the manifest, and would probably download things a few times a day, but if you run `make` repeatedly in a reasonably short period you won't get any updates and will just be paying for the download of the manifest. Up to y'all whether that seems worthwhile.
